docs/evaluation/code.md

```
@@ -178,6 +178,76 @@ all you need to do is replace `openhands` with `swe_agent` in the command above.
+  "livecodebench": { "pass@1": { "accuracy": 71.14 ... }},
+  "livecodebench-easy": { "pass@1": { "accuracy": 99.09 ... }},
+  "livecodebench-hard": { "pass@1": { "accuracy": 46.30 ... }},
+  "livecodebench-medium": { "pass@1": { "accuracy": 85.10 ... }}
+}
+```
+
+##### Advanced: Averaging Multiple Runs
+
+Due to variance between runs, you can automatically repeat the evaluation and average the results. To run the evaluation 3 times, for example, set the `--benchmarks` flag as follows:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)